ansible_community_steering_committee.rst

```diff
@@ -2,31 +2,28 @@
 Ansible Community Steering Committee
 ************************************
 
-The Steering Committee’s role is to provide continuity, guidance, and
-suggestions to the Ansible community, and to ensure delivery of the Ansible
-Community package. The Steering Committee should reflect the scope and
-breadth of the Ansible Community.
-
 Mission:
 ========
 
-The Steering Committee helps decide the technical direction of the
-Ansible project and is also responsible for avowal of new
-proposals (feature idea, new policy in collection world), new collection
-requests, empower contributors, maintainers & reviewers and other
-technical aspects regarding inclusion and packaging.
+The Steering Committee’s mission (hereinafter the Committee) is to provide continuity, guidance, and
+suggestions to the Ansible community, to ensure delivery and high quality of the Ansible package.
+
+The Committee helps decide the technical direction of the Ansible project and is responsible for approval of new
+proposals and policies in the community, package, and collections' world, new collection-inclusion requests,
+and other technical aspects regarding inclusion and packaging.
 
-Committee Member Responsibilities:
-==================================
+The Committee should reflect the scope and breadth of the Ansible Community.
 
-The members in the committee handle the below:
+The Committee Responsibilities:
+===============================
 
-- Design policy and procedure for the collection world.
-- Review collections for compliance with the policies. 
-- Help create our deliverables (community.general, community.network, the ``ansible`` tarball, docs website).
-- Ballot on approval changes to policy and new proposals.
-- Identify and train individuals from the community who are interested in contributing to Ansible.
-- Review collections submitted for inclusion in the Ansible package and decide whether include them or not.
+The committee:
+
+* Designs policies and procedures for the collection world; vote on approval changes to established policies and procedures.
```

Comment:
I'm not sure about this but should be we consider saying the "collection community" instead of the "collection world" throughout the document? Or do folks feel that is overusing or overloading the term "community"?

Reply:
I vote for using community since the steering committee doesn't establish policies etc for certified collections. Worth making the distinction here.

Reply:
Good catch! I've changed `collections` to `community collections` everywhere.